### PR TITLE
Add script for building Enterprise package on Travis

### PR DESCRIPTION
**Pull request by Lex-2008**
Note that currently Travis is not enabled for this repo, but will be enabled for several others

### CODE REVIEW

File: build-scripts/travis [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+./buildscripts/build-scripts/prepare-results
+
+ls output
+ls output/*
+
+if [ "$job" = "fast-php" ]; then
+	# clean build leftovers before going to php tests
+	./buildscripts/build-scripts/clean-buildmachine
+	cd mission-portal
+	job=php ./travis.sh ../output/*/*.deb
```
Comment (jimis) [inline build-scripts/travis]: what is this line doing?

Comment (Lex-2008) [inline build-scripts/travis]: it starts php unit tests for generated *.deb file using a script in mission-portal repo - I'm adding it in another PR right now, will post link here in a moment

Comment (jimis) [inline build-scripts/travis]: Great, thanks!

Comment (Lex-2008) [inline build-scripts/travis]: Related pull request is here: https://github.com/cfengine/mission-portal/pull/393

**Pull request merged**